src/veins/base/utils/util.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
```

Comment:
I'd rather cut back on the number of generic "miscellaneous utility" include files. If some construct is really needed, I'd rather have it in a dedicated include file that is only used where needed. If a construct is needed throughout the framework, it should be either in (or included by) `veins.h`.